sgpt/config.py

```diff
@@ -34,6 +34,9 @@
     "API_BASE_URL": os.getenv("API_BASE_URL", "default"),
     "PRETTIFY_MARKDOWN": os.getenv("PRETTIFY_MARKDOWN", "true"),
     "USE_LITELLM": os.getenv("USE_LITELLM", "false"),
+    "SHELL_INTERACTION ": os.getenv("SHELL_INTERACTION ", "true"),
+    "OVERWRITE_OS_NAME": os.getenv("OVERWRITE_OS_NAME", ""),
+    "OVERWRITE_SHELL_NAME ": os.getenv("OVERWRITE_SHELL_NAME", ""),
```

Comment:
```suggestion
    "SHELL_INTERACTION ": os.getenv("SHELL_INTERACTION ", "true"),
    "OVERWRITE_OS_NAME": os.getenv("OVERWRITE_OS_NAME", "default"),
    "OVERWRITE_SHELL_NAME ": os.getenv("OVERWRITE_SHELL_NAME", "default"),
```

Reply:
As stated above, I find the name "default" confusing for not using the var.
My personal preference goes to something like "", "no" or "no_overwrite".